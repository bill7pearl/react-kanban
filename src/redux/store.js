import { configureStore, applyMiddleware } from '@reduxjs/toolkit';
import thunk from 'redux-thunk';
import rocketReducer from './rockets/rockets';
import missionReducer from './missions/missions';

const store = configureStore(
  {
    reducer: {
      rockets: rocketReducer,
      missions: missionReducer,
    },
  },
  applyMiddleware(thunk),
);

export default store;
