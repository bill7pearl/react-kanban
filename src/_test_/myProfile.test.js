import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import store from '../redux/store';
import Myprofile from '../components/Myprofile';

describe('testing for rockets data', () => {
  it('test for component', () => {
    const test = render(
      <Provider store={store}>
        <Myprofile />
      </Provider>,
    );
    expect(test).toMatchSnapshot();
  });
});
