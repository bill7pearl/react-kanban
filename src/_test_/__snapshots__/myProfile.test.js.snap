// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing for rockets data test for component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="profile"
      >
        <div
          class="mission"
        >
          <h2
            class="top"
          >
            My Missions
          </h2>
          <ul
            class="li-li"
          >
            <li
              style="border: 1px solid grey;"
            >
              Telstar
            </li>
            <li
              style="border: 1px solid grey;"
            >
              SES
            </li>
            <li
              style="border: 1px solid grey;"
            >
              Asiasat
            </li>
            <li
              style="border: 1px solid grey;"
            >
              ABs
            </li>
          </ul>
        </div>
        <div
          class="rocket"
        >
          <h2
            class="top"
          >
            My Rockets
          </h2>
          <ul
            class="li-li"
          >
            <li
              style="border: 1px solid grey;"
            >
              Falcon9
            </li>
            <li
              style="border: 1px solid grey;"
            >
              Falcon Heavy
            </li>
            <li
              style="border: 1px solid grey;"
            >
              star ship
            </li>
          </ul>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="profile"
    >
      <div
        class="mission"
      >
        <h2
          class="top"
        >
          My Missions
        </h2>
        <ul
          class="li-li"
        >
          <li
            style="border: 1px solid grey;"
          >
            Telstar
          </li>
          <li
            style="border: 1px solid grey;"
          >
            SES
          </li>
          <li
            style="border: 1px solid grey;"
          >
            Asiasat
          </li>
          <li
            style="border: 1px solid grey;"
          >
            ABs
          </li>
        </ul>
      </div>
      <div
        class="rocket"
      >
        <h2
          class="top"
        >
          My Rockets
        </h2>
        <ul
          class="li-li"
        >
          <li
            style="border: 1px solid grey;"
          >
            Falcon9
          </li>
          <li
            style="border: 1px solid grey;"
          >
            Falcon Heavy
          </li>
          <li
            style="border: 1px solid grey;"
          >
            star ship
          </li>
        </ul>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
